### PR TITLE
PGMM-lumi lumi stability analysis

### PR DESCRIPTION
**Pull request by JosueMtzGar**
Information about V0 detector and new triggers has been added.

### CODE REVIEW

**Review by aalkin — CHANGES_REQUESTED** [review]
Please resolve merge conflicts to continue.

**Comment by JosueMtzGar** [comment]
I have done the relevant changes

File: PWGMM/Lumi/Tasks/lumiStability.cxx [hunk 1]
```diff
@@ -33,36 +33,121 @@ struct lumiStabilityTask {
...
+    histos.add("FV0/bcInTrigger", "In trigger per BC (FV0);BC in V0; counts", kTH1F, {axisFV0Triggger});
+    histos.add("FV0/bcSCenTrigger", "SCen trigger per BC (FV0);BC in V0; counts", kTH1F, {axisFV0Triggger});
+    histos.add("FV0/bcCenTrigger", "Out trigger per BC (FV0);BC in V0; counts", kTH1F, {axisFV0Triggger});
+    histos.add("FV0/bcSCenCenTrigger", "SCen and Cen trigger per BC (FV0);BC in V0; counts", kTH1F, {axisFV0Triggger});
   }
 
-  void process(aod::FT0s const& ft0s, aod::FDDs const& fdds, aod::BCsWithTimestamps const&)
+  bool checkAnyCoincidence(const std::vector<int>& channels)
+  {
+    std::map<int, int> channelPairs = {{0, 4}, {1, 5}, {2, 6}, {4, 7}};
```
Comment (aalkin) [inline PWGMM/Lumi/Tasks/lumiStability.cxx:68]: Map is probably too heavy for this, you can use `std::array<std::pair<int, int>, 4>` and you can also make it `constexpr`.

File: PWGMM/Lumi/Tasks/lumiStability.cxx [hunk 2]
```diff
@@ -33,36 +33,121 @@ struct lumiStabilityTask {
...
+        if (isCoinA && isCoinC) {
+          histos.fill(HIST("FDD/bcVCTriggerCoincidence"), localBC);
+        }
+      } // vertex and scentral true
     }   // loop over FDD events
 
     for (auto const& ft0 : ft0s) {
       auto bc = ft0.bc_as<BCsWithTimestamps>();
-      if (!bc.timestamp())
+      if (bc.timestamp() == false) {
```
Comment (aalkin) [inline PWGMM/Lumi/Tasks/lumiStability.cxx:148]: Timestamp is not boolean, you should compare it with 0 instead.

**Review by aalkin — APPROVED** [review]
Merging as is, but please consider my other comment for the next time.

**Comment by JosueMtzGar** [comment]
Alright, thanks!

**Pull request merged**